File: Amazon_LeetCode_Python_C++/ArrayString/771_jewels_and_stones.cpp
class Solution {
public:
    int numJewelsInStones(string jewels, string stones) {
        int ans = 0;
        set<char> sj;
        for (int i = 0; i < jewels.size(); i++) {
            if (!sj.count(jewels[i])) {
                sj.insert(jewels[i]);
            }
        }
        for (int i = 0; i < stones.size(); i++) {
            if (sj.count(stones[i])) {
                ans += 1;
            }
        }
        return ans;
    }
};

File: Amazon_LeetCode_Python_C++/ArrayString/771_jewels_and_stones.py
class Solution:
    def numJewelsInStones(self, jewels: str, stones: str) -> int:
        j = list(jewels)
        s = list(stones)
        ans  = 0
        for i, c in enumerate(stones):
            if c in j:
                ans += 1
                
        return ans
